pyvista/plotting/axes_actor.py

```diff
@@ -388,111 +745,391 @@ def tip_type(self) -> TipType:  # numpydoc ignore=RT01
+
+    @property
+    def x_axis_label(self) -> str:  # numpydoc ignore=RT01
+        """Text label for the x-axis.
+
+        This parameter is deprecated. Use `x_label` instead.
+
+        """
+        warnings.warn(
+            "Use of `x_axis_label` is deprecated. Use `x_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        return self.x_label
+
     @x_axis_label.setter
     def x_axis_label(self, label: str):  # numpydoc ignore=GL08
-        self.SetXAxisLabelText(label)
+        warnings.warn(
+            "Use of `x_axis_label` is deprecated. Use `x_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        self.x_label = label
 
     @property
-    def y_axis_label(self) -> str:  # numpydoc ignore=RT01
-        """Return or set the label for the Y axis.
+    def y_label(self) -> str:  # numpydoc ignore=RT01
+        """Text label for the y-axis.
 
         Examples
         --------
         >>> import pyvista as pv
         >>> axes = pv.Axes()
-        >>> axes.axes_actor.y_axis_label = 'This axis'
-        >>> axes.axes_actor.y_axis_label
+        >>> axes.axes_actor.y_label = 'This axis'
+        >>> axes.axes_actor.y_label
         'This axis'
 
         """
         return self.GetYAxisLabelText()
 
+    @y_label.setter
+    def y_label(self, label: str):  # numpydoc ignore=GL08
+        self.SetYAxisLabelText(label)
+
+    @property
+    def y_axis_label(self) -> str:  # numpydoc ignore=RT01
+        """Text label for the y-axis.
+
+        This parameter is deprecated. Use `y_label` instead.
+
+        """
+        warnings.warn(
+            "Use of `y_axis_label` is deprecated. Use `y_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        return self.y_label
+
     @y_axis_label.setter
     def y_axis_label(self, label: str):  # numpydoc ignore=GL08
-        self.SetYAxisLabelText(label)
+        warnings.warn(
+            "Use of `y_axis_label` is deprecated. Use `y_label` instead.",
+            PyVistaDeprecationWarning,
+        )
+        self.y_label = label
 
     @property
-    def z_axis_label(self) -> str:  # numpydoc ignore=RT01
-        """Return or set the label for the Z axis.
+    def z_label(self) -> str:  # numpydoc ignore=RT01
+        """Text label for the z-axis.
 
         Examples
         --------
         >>> import pyvista as pv
         >>> axes = pv.Axes()
-        >>> axes.axes_actor.z_axis_label = 'This axis'
-        >>> axes.axes_actor.z_axis_label
+        >>> axes.axes_actor.z_label = 'This axis'
+        >>> axes.axes_actor.z_label
         'This axis'
 
         """
         return self.GetZAxisLabelText()
 
+    @z_label.setter
+    def z_label(self, label: str):  # numpydoc ignore=GL08
+        self.SetZAxisLabelText(label)
+
+    @property
+    def z_axis_label(self) -> str:  # numpydoc ignore=RT01
+        """Text label for the z-axis.
+
+        This parameter is deprecated. Use `z_label` instead.
```

Comment:
For deprecated parameters and classes, please specify your removal policy as follows This will make it easier for us in the future to determine whether it is acceptable to remove (see [Deprecating Features or other Backwards-Breaking Changes](https://docs.pyvista.org/version/dev//extras/developer_notes.html#deprecating-features-or-other-backwards-breaking-changes)).
```suggestion
        .. deprecated:: 0.43.0
           This parameter is deprecated and has been completely removed in version 0.46.0.
           Use `z_label` instead.
```

Reply:
This should also be resolved by [473fca3](https://github.com/pyvista/pyvista/pull/5029/commits/473fca38bbe6a029037c20cc383f7cbef487380f)